bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/scheduling/Strand.java

```diff
@@ -50,20 +52,25 @@
 
 public class Strand {
 
+    private static AtomicInteger nextStrandId = new AtomicInteger(0);
+
+    private int id;
+    private String name;
+    private StrandMetadata metadata;
```

Comment:
Let's change to StrandData by moving name inside `StrandMetadata` class

Reply:
For workers strand name is read at runtime hence we cannot moved to Strand metadata class which is constant at compiler time. So I have keep the name as it is since they are actually metadata. Added the private vars and getters to StrandMetadata class